project/dexter-vs/Dexter/Core/CodeSnippet/Dexter C++/ReviewCommentCpp.snippet

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>Code Reivew Comment</Title>
```

Comment:
Code Reivew Comment -> Code Review Comment

The same typo is in ReviewComment.snippet